recipe.ephys-viz/meta.yaml

```diff
@@ -15,12 +15,19 @@ build:
     - test -n "$BUILT_FROM_TOPDIR" # error if not built from mountainlab-conda directory
     - tgzfile=$(npm pack)
     - npm install -g $tgzfile
+    - python -m pip install vdom jp_proxy_widget
```

Comment:
We can convert these pretty easily to use conda for dependencies. vdom just needs 'ipython' + 'jsonschema', and jp_proxy_widget just needs 'requests' (at least according to a cursory reading of their setup.py files). Then we can use the `--no-deps --ignore-installed` args to pip install

Reply:
It's not fatal if we let pip install these dependencies, instead. Conda tries to play nice with pip, so I think it will be OK.